kahuna/public/js/edits/metadataDiff.spec.js

```diff
@@ -20,15 +20,15 @@ describe('metadataDiff', () => {
   it("finds a changed string field in the presence of an array", () => {
     const initial = { field: "value", array1: ["value 2"] };
     const changed = { field: "changed", array2: ["value 2"] };
-    const expected = { field: "changed" };
+    const expected = { field: "changed", array1: "", array2: ["value 2"] };
```

Comment:
Shouldn't `array1` be absent from `expected` here?

Reply:
I think we `array1` should return an empty array. In any case, I’ve updated the tests.